### PR TITLE
Fix comments in @code blocks in lac headers

### PR DESCRIPTION
**Pull request by masterleinad**
Relates to #6814.

### CODE REVIEW

File: include/deal.II/lac/schur_complement.h [hunk 1]
```diff
@@ -129,36 +129,45 @@ DEAL_II_NAMESPACE_OPEN
...
+ * y = S_inv * rhs;
+ *
+ * // PackagedOperation that represents the condensed form of g
+ * auto rhs = condense_schur_rhs (A_inv,C,f,g);
+ *
+ * // Solve for y
+ * y = S_inv * rhs;
+ *
+ * // Compute x using resolved solution y
+ * postprocess_schur_solution (A_inv,B,y,f);
```
Comment (drwells) [inline include/deal.II/lac/schur_complement.h]: I think this line should begin with `x =`.

File: include/deal.II/lac/block_linear_operator.h [hunk 2]
```diff
@@ -171,7 +171,8 @@ block_back_substitution(
  * auto top_left_op = linear_operator(top_left);
  * auto bottom_right_op = linear_operator(bottom_right);
  * std::array<decltype(top_left_op), 2> operators {{top_left_op,
- * bottom_right_op}}; auto block_op = block_diagonal_operator (operators);
+ *                                                  bottom_right_op}};
```
Comment (drwells) [inline include/deal.II/lac/block_linear_operator.h]: sorry for not catching this before, but it looks like we truncated the `std::array` above.

File: include/deal.II/lac/schur_complement.h [hunk 3]
```diff
@@ -129,36 +129,45 @@ DEAL_II_NAMESPACE_OPEN
...
+ * const auto prec_A = PreconditionSelector<...>(A);
+ * const auto A_inv = inverse_operator<...>(A,prec_A);
+ * const auto S = schur_complement(A_inv,B,C,D);
+ *
+ * // D and S operate on same space
+ * const auto S_prec = PreconditionSelector<...>(D);
+ * const auto S_inv = inverse_operator<...>(S,...,prec_S);
+ *
+ * // Solve reduced block system
+ * // PackagedOperation that represents the condensed form of g
```
Comment (drwells) [inline include/deal.II/lac/schur_complement.h]: Another one I did not catch before: I think that the comment starting with `PackagedOperation` refers to the line containing `condense_schur_rhs`

Comment (drwells) [reply include/deal.II/lac/schur_complement.h]: Let me be more specific. Here is the original version:

```
 *    // Solve reduced block system
 *    auto rhs = condense_schur_rhs (A_inv,C,f,g); // PackagedOperation that represents the condensed form of g
 *    y = S_inv * rhs; // Solve for y
 *    x = postprocess_schur_solution (A_inv,B,y,f); // Compute x using resolved solution y
```

so the `y = ` line is duplicated.

File: include/deal.II/lac/schur_complement.h [hunk 4]
```diff
@@ -178,22 +187,29 @@ DEAL_II_NAMESPACE_OPEN
...
- *    x = postprocess_schur_solution (A_inv,B,y,f);
+ * // Construction of approximate inverse of Schur complement
+ * const auto A_inv_approx = linear_operator(preconditioner_A);
+ * const auto S_approx = schur_complement(A_inv_approx,B,C,D);
+ *
+ * // D and S_approx operate on same space
+ * const auto S_approx_prec = PreconditionSelector<...>(D);
+ *
+ * // Inner solver:
+ * // Typically limited to few iterations using IterationNumberControl
```
Comment (drwells) [inline include/deal.II/lac/schur_complement.h]: These two comment lines should be combined

File: include/deal.II/lac/block_linear_operator.h (hunk 2 above)
Comment (masterleinad) [reply include/deal.II/lac/block_linear_operator.h]: Hmm... Can you be more specific? I don't quite get the problem.

Comment (drwells) [reply include/deal.II/lac/block_linear_operator.h]: No, I misread the diff. This is fine. sorry

**Comment by drwells** [comment]
Since its just a documentation fix lets merge.

**Pull request merged**